packages/dds/tree/src/codec/codec.ts

```diff
@@ -252,9 +252,10 @@ export interface ICodecFamily<TDecoded, TContext = void> {
 /**
  * A version stamp for encoded data.
  *
+ * Strings are used for formats that are not yet officially supported.
```

Comment:
I think we should clarify that once stable, we don't start officially supporting string, but move them to a number:
```suggestion
 * Strings are used for formats that are not yet officially supported.
 * When such formats become officially supported/stable, they will be switched to using a number.
```